README.md

```diff
@@ -271,6 +271,15 @@ You must disable this feature by setting the option to `false`.
 To find more examples on how to use the configuration file please refer to the
 tests.
 
+#### Available locales
+
+Setting the `js_available_locales` option allows to generate translations not only
+for `I18n.available_locales` locales (which we use by default). Example:
```

Comment:
```suggestion
By specifying option `js_available_locales` with a list of locales, this list would be used instead of default `I18n.available_locales` to generate translations.  
Example:
```